client/gutenberg/extensions/publicize/editor.js

```diff
@@ -5,19 +5,50 @@
  *
  * Hooks into Gutenberg's PluginPrePublishPanel
  * to display Jetpack's Publicize UI in the pre-publish flow.
+ *
+ * It also hooks into our dedicated Jetpack plugin sidebar and
+ * displays the Publicize UI there.
+ */
+
+/**
+ * External dependencies
  */
+import { Fragment } from '@wordpress/element';
+import { PluginPrePublishPanel } from '@wordpress/edit-post';
+import { PanelBody } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
+import JetpackPluginSidebar from 'gutenberg/extensions/presets/jetpack/editor-shared/jetpack-plugin-sidebar';
 import PublicizePanel from './panel';
 import registerJetpackPlugin from 'gutenberg/extensions/presets/jetpack/utils/register-jetpack-plugin';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 
 export const name = 'publicize';
 
 export const settings = {
-	render: () => <PublicizePanel />,
+	render: () => (
+		<Fragment>
+			<JetpackPluginSidebar>
+				<PanelBody title={ __( 'Share this post' ) }>
+					<PublicizePanel />
+				</PanelBody>
+			</JetpackPluginSidebar>
+			<PluginPrePublishPanel
+				initialOpen={ true }
```

Comment:
Let's drop the redundant `={ true  }` while we're at it.

Reply:
👍 229f835d20bafd145a732f34845804d799e91d4d